test/ci/before_install.sh

```diff
@@ -24,5 +24,6 @@ case "$TRAVIS_OS_NAME" in
       -in codesign.csr -out codesign.cer
     openssl pkcs12 -export -in codesign.cer -inkey codesign.key -out codesign.p12 -password pass:12345
     security import codesign.p12 -k ~/Library/Keychains/login.keychain -P 12345 -T /usr/bin/codesign
+    brew install wine
```

Comment:
FYI, the last time I tried this I had to do some weird things so that the OSX CI builds didn't take forever to run and used a relatively recent version of Wine: https://github.com/electron-userland/electron-packager/blob/v7.1.0/test/ci/before_install.sh#L6-L11